tests/fv3_conf/control_run.IN

```diff
@@ -80,7 +80,8 @@ cp    @[INPUTDATA_ROOT]/${inputdir}/field_table_gfsv16 ./field_table
 cp    @[INPUTDATA_ROOT]/${inputdir}/*configure .
 
 if [ $NEW_DIAGTABLE != '' ]; then
-  cp    @[INPUTDATA_ROOT]/${inputdir}/$NEW_DIAGTABLE ./diag_table
+  #cp    @[INPUTDATA_ROOT]/${inputdir}/$NEW_DIAGTABLE ./diag_table
```

Comment:
This change is temporary until the new file `diag_table_gfsv16_thompson_extdiag` has been added to the input data directory.